### PR TITLE
Bumping version to 0.0.3

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "origin-web-catalogs",
```
Comment (spadgett) [inline package.json]: Just noticed this does not match bower.json

**Comment by jeff-phillips-18** [comment]
Thanks @spadgett lots of catalogs that should be catalog in there. Fixed.

File: package.json [hunk 2]
```diff
@@ -17,14 +17,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/origin/origin-web-catalogs.git"
+    "url": "https://github.com/origin/origin-web-catalog.git"
   },
-  "author": "Jeff Phillips, David Taylor",
+  "author": "Red Hat, Inc",
   "license": "MIT",
```
Comment (spadgett) [inline package.json]: @jeff-phillips-18 This should probably be Apache License 2.0 since the rest of OpenShift is

cc @jwforres

Comment (jwforres) [inline package.json]: Yes, openshift is Apache 2.0

Comment (jwforres) [inline package.json]: I'm not seeing a LICENSE file at all, can we go ahead and fix that too, grab the one in origin-web-console

**Pull request merged**